request_builder.rb

```diff
@@ -83,29 +83,32 @@ def send_post_request
     end
 
     if !params['filePath'].nil?
-      http, request = prepare_multipart_request(params)
+      http, request = self.prepare_multipart_request params
```

Comment:
I don't think this is general consensus, but I prefer to use `self` for any public method I can to remove any possible ambiguity regarding the method call.